app/controllers/miq_ae_class_controller.rb

```diff
@@ -2340,7 +2340,7 @@ def set_method_record_vars(miqaemethod)
                          @edit[:new][:data]
                        end
     miqaemethod.class_id = from_cid(@edit[:ae_class_id])
-    miqaemethod.embedded_methods = @edit[:new][:embedded_methods] if @edit[:new][:location] == 'inline'
+    miqaemethod.embedded_methods = @edit[:new][:location] == 'inline' ? @edit[:new][:embedded_methods] : []
```

Comment:
What does `@edit[:new][:embedded_methods]` contain for non-inline methods?  Does it always get initialized to an empty array?

If so this could just be:
`miqaemethod.embedded_methods = @edit[:new][:embedded_methods]`

Reply:
@gmcculloug `@edit[:new][:embedded_methods]` is only set if location is 'inline' in other cases it will be nil